libraries/core/src/morpheus/core/serialisation/json_reader.hpp

```diff
@@ -80,23 +98,22 @@ class MORPHEUSCORE_EXPORT JsonReader
     }
 
     /// Reads a integral type from the serialisation.
-    template<std::integral Interger> requires (not std::is_same_v<bool, Interger>)
+    template <std::integral Interger>
+    requires(not std::is_same_v<bool, Interger>)
     Interger read()
     {
         auto const [event, next] = getNext();
-        MORPHEUS_ASSERT((next->index() == 1) or (next->index() == 2));
```

Comment:
It does not make sense to asset something that is then validated bu runtime checks immediately after.